ajax/libs/dna.js/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "dna.js",
+  "autoupdate": {
+   "source": "git",
+   "target": "git://github.com/dnajs/dna.js.git",
+   "fileMap": [
+    {
+      "basePath":  "",
+      "files": [
+        "dna.min.js",
+        "dna.js"
+      ]
+    }
+  ]
+}
```

Comment:
yes sir, just noticed .  And should i squash the last two commits and make one commit or multiple commits are fine?